content/docs/started/workstation/getting-started-linux.md

```diff
@@ -0,0 +1,80 @@
++++
+title = "Kubeflow on Linux"
+description = "Install Kubeflow on Linux"
+weight = 2
++++
+
+<!--
+  TODO: Create a table that summarizes the options below, helping the user choose
+  more quickly
+-->
+
+## Introduction
+
+For Linux systems you have options for servers and desktops. The server options apply to
+the desktop as well.
+
+## Linux Server
+
+For linux servers you can install Kubeflow natively. This is perfect for
+linux hosts and virtual machines, such as VMs in OpenStack, VMware or public clouds like
+GCP, AWS and Azure.
+
+### MicroK8s
+
+[MicroK8s](https://microk8s.io) runs natively on most Linux distributions. It requires
+[snapd](https://snapcraft.io/docs/getting-started), which is pre-installed on Ubuntu
+and Ubuntu derivative operating systems.
+
+1. Install MicroK8s - `snap install microk8s --classic`
+2. Follow the getting started guid for Kubeflow on an
+[existing Kubernetes cluster](/docs/started/k8s/overview/).
+
+## Linux Desktop
+
+### Kubeflow Appliance
+
+A Kubeflow appliance is a virtual machine that has Kubeflow already installed. Once the
+necessary supporting software is installed no further installation steps are required.
+
+#### MiniKF
+
+MiniKF is a predefined virtual machine that installs onto VirtualBox through Vagrant.
+The only following applications are required to use MiniKF:
+
+- Install [Vagrant](https://www.vagrantup.com/downloads.html)
+- Install [Virtual Box](https://www.virtualbox.org/wiki/Downloads)
+
```

Comment:
Can you restore the MiniKF description as it was on the Getting Started page?

>MiniKF is a fast and easy way to get started with Kubeflow. It installs with just two commands and then you are up for experimentation, and for running complete Kubeflow Pipelines. It runs on all major operating systems (macOS, Windows, Linux).

Reply:
I thought I had selected the whole description, but GitHub's multi-line review doesn't seem to work as expected.

Reply:
I'm thinking of abbreviating this section further, drawing inspiration from the existing Kubernetes section. On that page a table is used, with each row providing a brief description of what the option is and then a pointer to the maintained page. Similar to that page, I'd like to avoid adjectives, and provide the complete set of instructions - ie in this scenario, that Vagrant and VirtualBox is required.